python/cdp/test/test_e2e.py

```diff
@@ -1859,6 +1859,31 @@ async def test_evm_local_account_sign_and_send_transaction(cdp_client):
 
 @pytest.mark.e2e
 @pytest.mark.asyncio
+async def test_use_network_evm_smart_account_e2e(cdp_client):
```

Comment:
test name doesn't need e2e
```suggestion
async def test_use_network_evm_smart_account(cdp_client):
```